services/app-web/src/components/Form/FieldTextarea/FieldTextarea.tsx

```diff
@@ -42,7 +42,7 @@ export const FieldTextarea = ({
                 <PoliteErrorMessage>{meta.error}</PoliteErrorMessage>
             )}
             {hint && (
-                <div aria-labelledby={id} className="usa-hint margin-top-1">
+                <div role="note" aria-labelledby={id} className="usa-hint margin-top-1">
```

Comment:
In the spirit of unifying out re-useable form components could you please add the `role=note` to the other hint handling as well please. We have  the exact same time of hint handling for FieldDropdown  FieldTextInput  and as well looks like.